test/reshape.jl

```diff
@@ -279,6 +279,13 @@ end
     @test_throws ArgumentError d1s[!, 1][1.0]
     @test_throws ArgumentError d1s[!, 2][1.0]
 
+    d1ss = stack(d1, [:a, :b], view=true)
+    @test d1ss[!, 1][[1,24]] == ["a", "b"]
+    d1ss = stack(d1, [:a, :b], view=true, vareltype=String)
+    @test d1ss[!, 1][[1,24]] == ["a", "b"]
```

Comment:
Also check that the result is a `PooledArray`?

Reply:
This is a `view=true` test, so it will be `RepeatedArray` in all cases.